include/cargo_loading_service/cargo_loading_service.hpp

```diff
@@ -35,43 +37,50 @@ class CargoLoadingService : public rclcpp::Node
 
 private:
   using ExecuteInParkingTask = in_parking_msgs::srv::ExecuteInParkingTask;
+  using InfrastructureCommand = v2i_interface_msgs::msg::InfrastructureCommand;
   using InfrastructureCommandArray = v2i_interface_msgs::msg::InfrastructureCommandArray;
-  using InParkingStatus = in_parking_msgs::msg::InParkingStatus;
+  using InfrastructureState = v2i_interface_msgs::msg::InfrastructureState;
   using InfrastructureStateArray = v2i_interface_msgs::msg::InfrastructureStateArray;
+  using InParkingStatus = in_parking_msgs::msg::InParkingStatus;
 
-  static constexpr uint8_t REQUESTING = 1;
-  static constexpr uint8_t ERROR = 2;
+  // constants
+  enum class CommandState : uint8_t { REQUESTING = 0b01, ERROR = 0b10 };
 
-  std::mutex mutex_cargo_loading_state_;
-  std::mutex mutex_parking_state_;
-  struct FacilityInfo {
-    std::string id;
-    bool approval;
-  } facility_info_;
-  int32_t aw_state_;
-  std::chrono::nanoseconds command_pub_sleep_time_;
-  double cargo_loading_command_pub_hz_;
+  static constexpr char COMMAND_TYPE[] = "eva_beacon_system";
 
-  // Callback group
-  rclcpp::CallbackGroup::SharedPtr callback_group_service_;
-  rclcpp::CallbackGroup::SharedPtr callback_group_subscription_;
+  // variable
+  std::string facility_id_;
```

Comment:
```suggestion
  uint8_t infra_id_;
```
facility vs infraで使い分けに意味があまりないと思いますので、主流なinfraのほうに修正します。